doc/releases/tentacle.rst

```diff
@@ -0,0 +1,395 @@
+  responses.
+* IAM policy evaluation now supports conditions ArnEquals and ArnLike,
+  along with their Not and IfExists variants.
+
+CephFS
+
+* Directories may now be configured with case-insensitive or normalized
+  directory entry names.
+* Modifying the FS setting variable "max_mds" when a cluster is unhealthy
+  now requires users to pass the confirmation flag (--yes-i-really-mean-it).
+* EOPNOTSUPP (Operation not supported) is now returned by the CephFS fuse
+  client for `fallocate` for the default case (i.e. mode == 0).
+
+Ceph
+----
+
+* Integrated SMB support Ceph clusters now support an smb manager module
+  that works similarly to the existing nfs subsystem. The new smb support
+  allows the Ceph cluster to automatically create Samba-backed SMB file
+  shares connected to CephFS. The smb module can configure both basic
+  Active Directory domain or standalone user authentication. The Ceph
+  cluster can host one or more virtual smb cluster which can be truly
+  clustered using Samba's CTDB technology. The smb module requires a
+  cephadm enabled Ceph cluster and deploys container images provided by
+  the samba-container project. The Ceph dashboard can be used to configure
+  smb clusters and shares. A new cephfs-proxy daemon is automatically
+  deployed to improve the scalibilty and memory usage when connecting
+  Samba to CephFS.
+
+CephFS
+------
+
+* Directories may now be configured with case-insensitive or normalized
+  directory entry names. This is an inheritable configuration making
+  it apply to an entire directory tree. For more information, see
+  https://docs.ceph.com/en/latest/cephfs/charmap/
+* Modifying the FS setting variable "max_mds" when a cluster is
+  unhealthy now requires users to pass the confirmation flag
+  (--yes-i-really-mean-it). This has been added as a precaution to tell the
+  users that modifying "max_mds" may not help with troubleshooting or recovery
+  effort. Instead, it might further destabilize the cluster.
+* EOPNOTSUPP (Operation not supported) is now returned by the CephFS
+  fuse client for `fallocate` for the default case (i.e. mode == 0) since
+  CephFS does not support disk space reservation. The only flags supported are
+  `FALLOC_FL_KEEP_SIZE` and `FALLOC_FL_PUNCH_HOLE`.
+
+Dashboard
+---------
+
+* There is now added support for NVMe/TCP (gateway groups, multiple
+  namespaces), multi-cluster management, oAuth2 integration, and enhanced
+  RGW/SMB features including multi-site automation, tiering, policies,
+  lifecycles, notifications, and granular replication.
+
+MGR
+---
+
+* Users now have the ability to force-disable always-on modules and the
+  removal of the restful and zabbix modules (both deprecated since 2020).
+  Note that the dashboard module's richer and better-maintained RESTful
+  API can be used as an alternative to the restful module, and the
+  prometheus module can be used as an alternative monitoring solution for
+  zabbix.
```

Comment:
Ditto.

Suggest

``restful`` module.  Grafana, the Prometheus time-series database, and Alertmanager are recommended as alternatives to Zabbix.